Cargo.toml

```diff
@@ -63,6 +63,20 @@ exclude = [
   "src/tools/x",
 ]
 
+# These lints are applied to many crates in the workspace. In practice, this is
+# all crates under `compiler/`.
+#
+# NOTE: rustc-specific lints (e.g. `rustc::internal`) aren't supported by
+# Cargo. (Support for them is possibly blocked by #44690 (attributes for
+# tools).) Those lints are instead specified for `compiler/` crates in
+# `src/bootstrap/src/core/builder/cargo.rs`.
+[workspace.lints.rust]
```

Comment:
Suggestion: actually, can you leave a quick FIXME pointed to #138195 about the tidy check too?

Reply:
I think #138195 is enough, we don't need a comment in the code as well. I will do the tidy check next week. If I add a comment here I'll just have to take it out again there.

Reply:
Sure, that's fine.